hamster_gtk/hamster_gtk.py

```diff
@@ -363,11 +373,15 @@ def get_db_config():
                 })
             return result
 
+        def get_autocomplete_activities_offset():
+            return cp_instance.getint('Frontend', 'autocomplete_activities_offset')
```

Comment:
When existing config is used, selfoss crashes on`backports.configparser.NoSectionError: No section: 'Frontend'`